modules/core/src/viewports/globe-viewport.ts

```diff
@@ -59,37 +62,39 @@ export type GlobeViewportOptions = {
 };
 
 export default class GlobeViewport extends Viewport {
-  // @ts-ignore
-  longitude: number;
-  // @ts-ignore
-  latitude: number;
-  resolution: number;
+  longitude!: number;
+  latitude!: number;
+  resolution!: number;
 
   constructor(opts: GlobeViewportOptions = {}) {
     const {
       latitude = 0,
       longitude = 0,
       zoom = 0,
-      nearZMultiplier = 0.1,
-      farZMultiplier = 2,
+      nearZMultiplier = 0.5,
```

Comment:
These changes are from [another PR](https://github.com/visgl/deck.gl/pull/9201), they are appearing here because I based my PR off the other PR and that had a force push which has confused github